src/DoocsPVFactory.cc

```diff
@@ -52,19 +54,19 @@ namespace ChimeraTK {
     // "._HIST", which also has to fit into the 64 characters
     if(propertyDescription.name.length() > 64 - 6) {
       std::cerr << "WARNING: Disabling history for " << processArray->getName() << ". Name is too long." << std::endl;
-      doocsPV.reset(new DoocsProcessScalar<DOOCS_PRIMITIVE_T, DOOCS_T>(
-          propertyDescription.name.c_str(), _eqFct, processArray, _updater));
+      doocsPV.reset(
+          new DoocsProcessScalar<DOOCS_PRIMITIVE_T, DOOCS_T>(propertyDescription.name, _eqFct, processArray, _updater));
     }
     else {
       if(propertyDescription.hasHistory) {
         // version with history: EqFtc first
         doocsPV.reset(new DoocsProcessScalar<DOOCS_PRIMITIVE_T, DOOCS_T>(
```

Comment:
use boost::make_shared

Reply:
I don't want to open that rabbit how right now. The code is full of that, my change merely removed an extraneous c_str() call...